components/dashboard/src/user-settings/SSHKeys.tsx

```diff
@@ -210,18 +211,17 @@ export default function SSHKeys() {
                 ) : null}
             </div>
             {dataList.length === 0 ? (
-                <div className="bg-gray-100 dark:bg-gray-800 rounded-xl w-full h-96">
-                    <div className="pt-28 flex flex-col items-center w-112 m-auto">
-                        <Heading2 color="light" className="text-center pb-3 text-gray-500 dark:text-gray-400">
-                            No SSH Keys
-                        </Heading2>
-                        <Subheading className="pb-6">
+                <EmptyMessage
```

Comment:
**[pr-rider] 😄** - Noticed the alignment was all off here, so moved it over to use the new component for rendering these views.